widget/embedded/src/components/NotificationContent/NotificationContent.styles.ts

```diff
@@ -0,0 +1,19 @@
+import { styled } from '@rango-dev/ui';
+
+export const Container = styled('div', {
```

Comment:
![image](https://github.com/rango-exchange/rango-client/assets/120931880/7c46db0a-5d63-4a35-9b66-b88b412f892c)
please fix background color for dark theme

Reply:
I will check this